data/countries/CA.yaml

```diff
@@ -108,6 +108,15 @@ holidays:
             name:
               en: Family Day
               fr: Fête de la famille
+            active:
+              - to: 2019
+          # @source https://www.statutoryholidays.com/bc.php
```

Comment:
How about https://news.gov.bc.ca/releases/2018PREM0018-000188 and/or https://globalnews.ca/news/4017192/b-c-s-family-day-moved-3rd-week-february-2019/ as alternate sources? Those are more primary sources and statutoryholidays.com appears to be a data aggregator.

Reply:
Yes that's better. Will update to use that links. Thanks for the research.